README.md

```diff
@@ -215,8 +215,8 @@ The `liberty-plugin-archetype` is used to generate a basic multi-module project
     mvn archetype:generate \
         -DarchetypeGroupId=io.openliberty.tools \
         -DarchetypeArtifactId=liberty-plugin-archetype \
-        -DarchetypeVersion=3.2.2 \
-        -DlibertyPluginVersion=3.2 \
+        -DarchetypeVersion=3.3-M3-SNAPSHOT \
+        -DlibertyPluginVersion=3.3-M3-SNAPSHOT \
```

Comment:
I don't think we should use a milestone release in our archetypes. Shouldn't this be the last non-milestone release 3.2.2? Once 3.3 is out we can update to that.

Reply:
Good point. Running the samples with 3.2.2 won't pull in the latest archetype changes, but we can update the doc with 3.3 once its out.